### DISCUSSION TITLE
blog/microservice-architecture-in-aspnet-core/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#30] [General]
# blog/microservice-architecture-in-aspnet-core/

Let's go through a popular way of Building Applications - Microservice Architecture in ASP.NET Core. In this detailed article, we will try to understand what really Microservice architecture is. , How does it compare to the traditional way of building applications? And also some advanced concepts like API Gateways with Ocelot, Unifying several Microservices, Health Checks, and much more.

https://codewithmukesh.com/blog/microservice-architecture-in-aspnet-core/

**Comment by nzmike99** [comment]
Hey Mukesh, just wanted to take the time to thank you for this post, it really helped me understand more about how to approach a microservices API and implement an API gateway.   However, this article is obviously now quite old now so if you were to transfer this API to Azure hosting would you still implement a gateway API (using Ocelot() or would you use something like Azure API Management?  Would love to hear what you think!
